### ISSUE TITLE
 option " --charging " not working in ixpeobssim 

### ISSUE THREAD

**Issue by lucabaldini** [lucabaldini/ixpeobssim#619]
Charging parameters files are missing.

When launching a simulation with the option “ --charging True “, the following error message is obtained:

xpobssim.py --configfile ixpeobssim/config/toy_point_source.py --duration 10000 --charging True

>>> Input file /.../ixpeobssim/ixpeobssim/caldb/bcf/chrgparams/ixpe_vanilla_d1_chrgparams.fits does not exists

**Comment by lucabaldini** [comment]
Good catch. The file is there in the repo, but lives in a slightly different place, after a refactoring of the pseudo-caldb to bring it in line with the real one:

```
ixpeobssim/caldb/ixpe/gpd/bcf/chrgparams/ixpe_vanilla_d1_chrgparams.fits
```

Will fix this.

**Comment by lucabaldini** [comment]
This is fixed on the `restore_missing_files` branch.

**Comment by lucabaldini** [comment]
Shipped in 29.1.1
